release-schema.json

```
@@ -109,7 +109,7 @@
         },
         "operatorRevenueShare": {
           "title": "Operator revenue share",
-          "description": "The percentage of revenue from the sale of tickets allocated to the supplier.",
+          "description": "The percentage of revenue from the sale of tickets allocated to the supplier, expressed as a decimal proportion between 0 and 1.",
```

Since the field name doesn't use "percentage", we can drop the term entirely.

```suggestion
          "description": "The proportion of revenue from the sale of tickets allocated to the supplier, expressed as a decimal between 0 and 1.",
```